lib/actions/grants/authorization_code.js

```diff
@@ -25,6 +25,10 @@ module.exports.handler = function getAuthorizationCodeHandler(provider) {
 
     // PKCE check
     if (pkce && (ctx.oidc.params.code_verifier || code.codeChallenge)) {
+      if (!pkce.supportedMethods.includes(code.codeChallengeMethod)) {
+        ctx.throw(new InvalidGrantError('challenge method is not supported'));
+      }
+
```

Comment:
In what case would this codepath be executed? Code is only pushed to the DB by regular path (non-tests) after the exact same validation, having it again here is redundant.